rl4j/rl4j-core/src/main/java/org/deeplearning4j/rl4j/learning/sync/qlearning/QLearning.java

```diff
@@ -45,8 +45,11 @@
 public abstract class QLearning<O extends Encodable, A, AS extends ActionSpace<A>>
                 extends SyncLearning<O, A, AS, IDQN> {
 
-    @Getter
-    final private IExpReplay<A> expReplay;
+    // FIXME Changed for refac
+    // @Getter
+    // final private IExpReplay<A> expReplay;
+    @Getter @Setter
+    private IExpReplay<A> expReplay;
```

Comment:
Can we make this package private instead? That way we can set it in unit tests without exposing it to anything else.